src/classes/HiFiAudioAPIData.ts

```diff
@@ -107,6 +107,7 @@ export class HiFiAudioAPIData {
     position: Point3D;
     orientationEuler: OrientationEuler3D;
     orientationQuat: OrientationQuat3D;
+    volumeThreshold: number;
```

Comment:
I'm not in love with this variable name, but I'm having trouble coming up with a better one. :-) I immediately assumed that `volumeThreshold` was the MAXIMUM volume that a user connection could have, rather than (effectively) the minimum volume. Could this be considered to be the `noiseFloor`? Would that be confusing? (cc @kencooke )

Reply:
FYI and FWIW, not being immersed in the field: 
- To me naively, "limit" could be either floor or ceiling, but to refer to a maximum or ceiling I would use "limit", rather than "threshold". (I would take threshold to mean a floor or minimum accepted value.)
- Threshold is the term used in the mixer and the gate, although the word "volume" is not used. My thinking (such as it is) in adding the term "volume" is to pair it with the units that come back from the volume subscription, rather than -1 to 1, or a 16 bit int, etc.
- To me, a "noise floor" would be the lowest amount of noise one can hear. We are talking here about a "signal floor", or a "noise ceiling". (But alas, I don't think that "noise ceiling" would be very clear either. Argh.)

Reply:
Maybe "lowVolumeThreshold"? I definitely don't feel strongly here -- you do a good job of explaining it in the comments.

Reply:
Well, there is a loooooong history (in audio recording/broadcasting industry) of the activation point of a dynamic range processing tool being called a gate/compressor/limiter `threshold`. I'd prefer to explain the threshold in layman terms, than to make up a new and unconventional name for it.

Reply:
I don't have a preference between volumeThreshold, lowVolumeThreshold, or volumeGateThreshold. I'm good with any of them.